src/app/(main)/pokedex/_components/GachaCardItem.module.css

```
@@ -0,0 +1,20 @@
+@keyframes cardShine {
```

Stylelint (17.11.0)

[error] Expected keyframe name "cardShine" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)